Pipfile

```diff
@@ -42,7 +53,7 @@ coverage = ">=4.4.1"
 isort = ">=4.3"
 mock = ">=2.0.0"
 pydocstyle = ">=2.0.0"
-pytest = ">=3.3.1"
+pytest = ">=3.3.1,<4.0.0"
```

Comment:
why we have pytest twice? :)

Reply:
pytest-cov which is specified in this section installs pytest, if pytest is not duplicated in [packages] section,  then Pipenv will install version >=4.0 which is too new. 
Why pytest-cov is duplicated in both sections, I am not sure as it's your commit. :)